spec/services/google/client_service_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::ClientService, type: :service do
+  context 'when querying a simple keyword' do
+    it 'returns an HTTParty Response', vcr: 'google_search' do
+      result = described_class.new(keyword: FFaker::Lorem.word).call
+
+      expect(result).to be_an_instance_of(HTTParty::Response)
+    end
+
+    it 'queries Google Search', vcr: 'google_search' do
+      path = described_class.new(keyword: FFaker::Lorem.word).call.request.path
+
+      expect(path.to_s).to start_with('https://www.google.com/search')
```

Comment:
I think it can be used `Google::ClientService.BASE_SEARCH_URL` instead of checking hard coded value
```suggestion
      expect(path.to_s).to start_with(Google::ClientService.BASE_SEARCH_URL)
```

Reply:
Nice catch! @junan, it made me realized that google redirects to www.google.com and so I've updated the `BASE_SEARCH_URL` so that it includes the www.  ;-)